servers/physics/physics_server_sw.cpp

```diff
@@ -487,7 +487,8 @@ void PhysicsServerSW::body_set_space(RID p_body, RID p_space) {
 
 	while (body->get_constraint_map().size()) {
 		RID self = body->get_constraint_map().front()->key()->get_self();
-		ERR_CONTINUE(!self.is_valid());
+		if (!self.is_valid())
```

Comment:
Mi intent actually was to squash them all on approval to avoid flooding the history. But if you find it more useful to keep a certain degree of separation I can squash in groups by a closely related criterium, such as the two you pointed out.